package.json

```diff
@@ -121,9 +123,32 @@
             "eslint cypress --fix"
         ]
     },
-    "browserslist": [
-        "> 0.5%, last 2 versions, Firefox ESR, not dead, IE 11"
-    ],
+    "browserslist": {
+        "production": [
+            "defaults",
+            "supports es6-module",
+            "chrome > 62",
+            "firefox > 59",
+            "ios_saf >= 10.3",
+            "opera > 50",
+            "not chrome 61",
+            "not opera 48",
+            "not chrome 62",
+            "not opera 49"
```

Comment:
this is slightly ugly
and there may be a better way of doing it
but we're balancing saying "es6 support" and "navigator.webdriver" support